script/database_role/README.md

```diff
@@ -0,0 +1,20 @@
+# Terraform Module: DB SQL Script - Create readonly role
+
+Run SQL script in DB to create DB role(s).
+This script will be run on the machine which run this TF script.
```

Comment:
NIT: shall we add that you need a SSH key locally that is allowlisted on the Bastion Host?